### PR TITLE
Added code samples for stored infoTypes.

### PR DESCRIPTION
**Pull request by mwdaub**
The DLP team is in the process of expanding code samples to include features that we have launched since the initial samples were written.

This pull request adds code samples for the recently launched stored infoTypes feature. The samples were modeled after triggers.py and adapted to stored infoTypes. Note that this does not include code samples for actually using them - those will be added later.

### CODE REVIEW

**Comment by mwdaub** [comment]
Please take a look.

File: dlp/stored_info_types.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        'big_query_field': {
+            'table': {
+                'project_id': bq_input_project_id,
+                'dataset_id': bq_input_dataset_id,
+                'table_id': bq_input_table_id,
+            },
+            'field': {'name': bq_input_table_field},
+        }
+    }
+    create_stored_info_type(project, dictionary_config,
```
Comment (gguuss) [inline dlp/stored_info_types.py]: As alternative formatting, you could avoid the hanging indent:

```python
    create_stored_info_type(
        project, dictionary_config, stored_info_type_id=stored_info_type_id,
        display_name=display_name, description=description)
```

File: dlp/stored_info_types_test.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Copyright 2017 Google Inc.
```
Comment (gguuss) [inline dlp/stored_info_types_test.py]: If this is a new file, add this as 2019 instead of 2017

File: dlp/stored_info_types_test.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+    # Delete the files.
+    for blob in blobs:
+        blob.delete()
+
+    # Attempt to delete the bucket; this will only work if it is empty.
+    bucket.delete()
+
+
+def test_create_list_and_delete_stored_info_type(bucket, capsys):
```
Comment (gguuss) [inline dlp/stored_info_types_test.py]: Probably fine as-is but consider separating the create/delete and create/list/delete tests.

**Comment by andrewferlitsch** [comment]
from mcdaub: Hi Andrew, I originally submitted that PR in 2018 and then changed teams in early 2019. Because nobody responded to my PR request for over a month, I stopped working on it and no longer have the bandwidth to work on it since my new team is not associated with the code. You can either close the PR or ask my old team to open a new one with the same changes.

**Pull request closed without merging**